u8timeseries/timeseries.py

```diff
@@ -20,35 +19,34 @@
 
 class TimeSeries:
     def __init__(self,
-                 series: pd.Series,
-                 confidence_lo: Optional[pd.Series] = None,
-                 confidence_hi: Optional[pd.Series] = None,
+                 series: Union[pd.Series, pd.DataFrame],
                  freq: Optional[str] = None,
                  fill_missing_dates: Optional[bool] = True):
         """
-        A TimeSeries is an object representing a univariate time series, and optional confidence intervals.
+        A TimeSeries is an object representing a time series.
 
         TimeSeries are meant to be immutable.
 
         Parameters
         ----------
         series
-            The actual time series, as a pandas Series with a proper time index.
-        confidence_lo
-            Optionally, a Pandas Series representing lower confidence interval.
-        confidence_hi
-            Optionally, a Pandas Series representing upper confidence interval.
+            The actual time series, as a pandas Series or DataFrame with a proper time index.
         freq
             Optionally, a string representing the frequency of the Pandas Series. When creating a TimeSeries
             instance with a length smaller than 3, this argument must be passed.
         fill_missing_dates
             Optionally, a boolean value indicating whether to fill missing dates with NaN values
             in case the frequency of `series` cannot be inferred.
         """
+        # create dataframes
+        if (isinstance(series, pd.Series)):
```

Comment:
doing just DataFrame representation would help to avoid those considerations ;]

Reply:
done